### PR TITLE
Fix link-in-bio styles.

### PR DESCRIPTION
**Pull request by allilevine**
Issue raised here: p1689919397847559-slack-C9EJ7KSGH

## Proposed Changes

* Add the `link-in-bio` class to this giant list of style corrections. If these are needed for every flow, we should make them less specific.
* Fix font for all expanding textareas.

## Testing Instructions

* Go to /setup/link-in-bio/linkInBioSetup

Before | After
----- | -----
Screen Shot 2023-07-21 at 1 56 17 PM | Screen Shot 2023-07-21 at 1 55 56 PM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/global.scss [hunk 1]
```diff
@@ -77,6 +77,7 @@ button {
 .domains,
 .patterns,
 .link-in-bio-post-setup,
+.link-in-bio,
```
Comment (allilevine) [inline client/landing/stepper/declarative-flow/internals/global.scss:81]: Adding this to cover /setup/link-in-bio/linkInBioSetup

File: client/landing/stepper/declarative-flow/internals/steps-repository/components/setup-form/style.scss [hunk 2]
```diff
@@ -22,7 +22,8 @@
 	}
 
 	label,
-	textarea {
+	textarea,
+	.expanding-area textarea {
```
Comment (allilevine) [inline client/landing/stepper/declarative-flow/internals/steps-repository/components/setup-form/style.scss:26]: /setup/link-in-bio/linkInBioSetup has a specific `.expanding-area` textarea class that gets overwritten by another font family. I don't see it testing locally, but I see it on calypso.live.

**Comment by lsl** [comment]
This looks fixed in prod. Maybe someone else picked it up?

**Comment by allilevine** [comment]
> This looks fixed in prod. Maybe someone else picked it up?

I'm not seeing the fix on prod or staging. 🤔 I'm testing in Firefox and Chrome.

**Comment by lsl** [comment]
The first step is fine, the second one is not :facepalm:

**Review by nuriapenya — CHANGES_REQUESTED** [review]
The broken alignment seems to be fixed, but I still see issues with the styles of the buttons. In both Intro and Setup pages, the primary CTA still has the wrong values:

Perhaps this is more related to: https://github.com/Automattic/dotcom-forge/issues/3163

Screenshot 2023-07-25 at 08 10 33
Screenshot 2023-07-25 at 08 10 42

**Comment by nuriapenya** [comment]
Another thing I spot on the Launchpad screen in the testing link, is that the Header logo is repeated. Not happening in production. 

Screenshot 2023-07-25 at 08 23 56

**Comment by lsl** [comment]
> Another thing I spot on the Launchpad screen in the testing link, is that the Header logo is repeated. Not happening in production.

Are we loading the header twice and hiding one? (I think I've seen this before somewhere, sorry couldn't find a link). I don't think this will be an issue in production, from memory it was some weird css ordering issue that only took place in development.

> but I still see issues with the styles of the buttons. 

This should be fixed by https://github.com/Automattic/wp-calypso/pull/79708.

**Review by lsl — APPROVED** [review]
LGTM

**Comment by paulopmt1** [comment]
> is that the Header logo is repeated. Not happening in production.

I was also seeing that in both dev and staging. Since it wasn't present on trunk, I fixed [with this commit](https://github.com/Automattic/wp-calypso/pull/79756/commits/d3c7e6fe2073a7c43c86d099548b620603dcb8ab).

**Review by paulopmt1 — APPROVED** [review]
Nice work @allilevine, it's working as expected!

**Comment by allilevine** [comment]
Thank you! 🙌 Looks like everything except the button color has been resolved in production.

**Comment by paulopmt1** [comment]
> Thank you! 🙌 Looks like everything except the button color has been resolved in production.

I'll monitor https://github.com/Automattic/wp-calypso/pull/79708 and if it does not solve it, will create a fix for that.

**Pull request merged**